install-eaf.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
Can keep sh this place?

I remember some contributor change from bash to sh.

Reply:
@chenzaichun Check this.

Reply:
bash is installed on most of Linux distro and is the default shell. What shell should it support, ksh, csh, tcsh?

Reply:
Check https://github.com/manateelazycat/emacs-application-framework/issues/631

Reply:
Thanks. I'll update it later. by the way, As i know, $() is not support by all some shell, for example, csh is not support it.

Reply:
I'll remove array usage and command -v calling like #631.

Reply:
> I'll remove array usage and command -v calling like #631.

Nice ;)